packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -12,6 +12,35 @@ import { ChartBottomWidgets, ChartToolbarWidgets, ChartTopWidgets, DigitsWidget
 import FormLayout from '../Components/Form/form-layout.jsx';
 import AllMarkers from '../../SmartChart/Components/all-markers.jsx';
 
+function masterData() {
```

Comment:
Can we add a comment on why we are using this mocking function? Maybe also a TODO on how to make this proper with "real" data instead of just mocks in the future. SSR on the horizon?